apps/indexer-coordinator/src/payg/payg.sync.service.ts

```diff
@@ -169,11 +154,8 @@ export class PaygSyncService implements OnApplicationBootstrap {
   }
 
   async syncOpen(id: string, consumer: string, agent: string, price: string) {
-    let channel = await this.paygService.channel(id);
-    if (!channel) {
-      const channelState = await this.paygService.channelFromContract(id);
-      channel = await this.paygService.saveChannel(id, channelState, price, agent);
-    }
+    const channel = await this.paygService.channel(id);
```

Comment:
syncOpen is only called from the event listener. We can trust the input so no need to retrieve details again from contract.